CHANGELOG.md

```diff
@@ -33,17 +39,22 @@
 - xen-api 4.0.0
 - @vates/nbd-client 3.0.2
 - @xen-orchestra/xapi 6.0.0
-- @xen-orchestra/backups 0.49.0
 - @xen-orchestra/backups-cli 1.0.19
 - @xen-orchestra/cr-seed-cli 1.0.2
 - @xen-orchestra/immutable-backups 1.0.6
-- @xen-orchestra/proxy 0.28.2
 - xapi-explore-sr 0.4.4
 - xo-cli 0.28.0
-- xo-server 5.144.0
 - xo-server-backup-reports 0.19.0
-- xo-web 5.146.0
 - @xen-orchestra/web patch
```

Comment:
Not related to this PR, but should not have been patch.

Also, we probably needs to do another release of `@xen-orchestra/web` since there were some changes, no?